docs/guides/setup-channel.mdx

```diff
@@ -85,20 +84,9 @@ Available plans:
     * The `channel` parameter allows you to distinguish between different account types. When initializing the account, IBM Cloud is the default account used if have saved credentials for an IBM Quantum Platform and an IBM Cloud account.
 
     <Admonition type="caution">
-      Account credentials are saved in plain text, so only do so if you are using a trusted device.
+      Account credentials are saved in plain text. To more securely manage your token, it is recommended to use an environment variable; see the [Securely manage your token](#securely-manage) section for more information.
     </Admonition>
 
-    <Admonition type="note">
-    IBM Cloud is the default account used if you don't specify a different channel or account name.
-    </Admonition>
-
-    <CodeAssistantAdmonition
```

Comment:
This is removed because it's not secure, right? Indeed, you should not send a credential to Qiskit Code Assistant

Reply:
Removing this was an accident! I will restore the one above but delete the code assistant one -  @Eric-Arellano verifying I've got it right?

Reply:
Yeah, that sounds good. Thanks!